upf/upf_api.c

```diff
@@ -1061,6 +1061,78 @@ vl_api_upf_tdf_ul_table_add_t_handler (vl_api_upf_tdf_ul_table_add_t * mp)
   REPLY_MACRO (VL_API_UPF_TDF_UL_TABLE_ADD_REPLY);
 }
 
+/* API message handler */
+static void
+vl_api_upf_ueip_pool_nwi_add_t_handler (vl_api_upf_ueip_pool_nwi_add_t * mp)
+{
+  int rv = 0;
+  upf_main_t *sm = &upf_main;
+  vl_api_upf_ueip_pool_nwi_add_reply_t *rmp = NULL;
+
+  u8 *nwi_name = mp->nwi_name;
+  u8 *nwi_name_vec = 0;
+  u8 *identity = mp->identity;
+  u8 *identity_vec = 0;
+
+  do
+    {
+      if (mp->identity_len > 64 || mp->nwi_name_len > 64)
+	{
+	  rv = VNET_API_ERROR_INVALID_VALUE;
+	  break;
```

Comment:
`goto` is often frowned upon, but not in VPP code. So in order to be consistent with the rest of the code, we either need to switch to this `do { ... } while(0)` construct elsewhere, or use `goto` here like in other API handlers (granted, some of them use `out` and some of them use `reply` label, but that IMO is a lesser inconsistency, even though also worth fixing at some point, maybe).
So I'd just use a `reply:` label and a `goto` for this case